### PR TITLE
Web pack setup and List structure

### PR DESCRIPTION
**Pull request by Athaliah8**
## In This Repo

- Set up a new project with Webpack .

- Created an index.html file and write your HTML markup here.

- Created an index.js file and set an array of some simple to-do tasks.

- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.

- Rendered the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

- Created a style.css and set rules for the To-Do List.

### CODE REVIEW

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Athaliah8, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Everything is good but when I try to run the project on the machine by using `npm start` then it is not running because you don't have a script to run the project. please write the script on `package.json` file so it won't give any errors while running the project.
- [ ] You should stick to the design provided by microverse. please do so

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack"
   },
+  "scripts": {
+    "start": "node index.js"
+},
```
Comment (DeliceLydia) [inline package.json]: - [ ] I can't run the server I suggest that you use `webpack` to run it
 ```
"scripts": {
    "start": "webpack serve --open",
    "build": "webpack"
  },
  ```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Athaliah8, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @Athaliah8 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
No linter errors. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
Webpack dev server is set up and running :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**